unlock-app/src/components/interface/layouts/AppLayout.tsx

```diff
@@ -177,110 +177,108 @@ export const AppLayout = ({
   }
 
   return (
-    <>
```

Comment:
I just removed the `<>` around the `<div>`... but of course the diff looks weird!